interview-questions.md

```diff
@@ -10,34 +10,87 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 Your answer:
 
+You need to define the relationship from the parent model Cohort to the child model Student. You can do this by linking them with belongs_to (ex. student belongs_to cohort). Then you will need to generate a migration to add_user_id_to_student user_id:integer. This will add a column to your student table called user_id: that will hold the foreign key. 
```

Comment:
Remember that the foreign key name will always start with the name of the parent model. In this case, the foreign key would be `cohort_id`!